components/org.wso2.bfsi.identity.extensions/src/main/java/org/wso2/bfsi/identity/extensions/internal/IdentityExtensionsDataHolder.java

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.bfsi.identity.extensions.internal;
+
+import org.wso2.bfsi.consent.management.common.config.ConsentManagementConfigurationService;
+import org.wso2.bfsi.consent.management.common.util.ConsentManagementConstants;
+import org.wso2.bfsi.identity.extensions.auth.extensions.request.validator.BFSIRequestObjectValidator;
+import org.wso2.bfsi.identity.extensions.auth.extensions.response.handler.BFSIResponseTypeHandler;
+import org.wso2.bfsi.identity.extensions.claims.BFSIClaimProvider;
+import org.wso2.bfsi.identity.extensions.util.IdentityCommonUtils;
+import org.wso2.carbon.identity.application.mgt.ApplicationManagementService;
+import org.wso2.carbon.identity.oauth2.OAuth2Service;
+import org.wso2.carbon.identity.openidconnect.ClaimProvider;
+import org.wso2.carbon.identity.openidconnect.RequestObjectService;
+import org.wso2.carbon.user.core.service.RealmService;
+
+import java.util.Map;
+
+/**
+ * Data Holder for Identity Extensions.
+ */
+public class IdentityExtensionsDataHolder {
+
+    private static volatile IdentityExtensionsDataHolder instance;
+    private static ApplicationManagementService applicationManagementService;
+    private static ConsentManagementConfigurationService configurationService;
+    Map<String, Object> configurationMap;
```

Comment:
private

Reply:
FIxed in https://github.com/wso2-extensions/bfsi-consent-management/pull/6/commits/62253df1607d32bdebcd905b7140f43984f3c36a